src/Core__Option.resi

```diff
@@ -89,9 +115,9 @@ Option.getUnsafe(Some(3)) == 3
 Option.getUnsafe(None) // Raises an error
 ```
 
-## Exceptions
+## Important
 
-- This is an unsafe operation, it assumes `value` is neither `None` nor `Some(None(...)))`
+This is an unsafe operation, it assumes `value` is neither `None`, `Some(None))`, `Some(Some(None))` etc.
```

Comment:
Add `-`, as you did for [Option PR](https://github.com/rescript-association/rescript-core/pull/67/files#diff-d723610a829b84897b010442ec73c5ed6d4aeb84e071be7f3684914fec91c048R130)

```suggestion
- This is an unsafe operation, it assumes `value` is neither `None`, `Some(None))`, `Some(Some(None))` etc.

```